compiler_opt/rl/compilation_runner.py

```diff
@@ -80,14 +80,23 @@ def __exit__(self, exc, value, tb):
     pass
 
 
-def get_workdir_context():
+def get_workdir_context(keep_temps: Optional[str] = None):
```

Comment:
nit: s/keep_temps/explicit_temps_dir. Otherwise it sounds like a boolean (yes, documentation explains, but why call the object bicycle when it's a car lol)